src/httpc.c

```diff
@@ -173,9 +173,10 @@ httpc_request_delete(struct httpc_request *req)
 int
 httpc_set_header(struct httpc_request *req, const char *fmt, ...)
 {
+	static __thread char header[TT_STATIC_BUF_LEN*8];
```

Comment:
I think it is better to provide explicit value (8192) here. You can define it at the top of the file and use here.

Reply:
I add new define HTTP_HEADER_LEN in httpc.h for this